src/utils/rounded.jsx

```diff
@@ -0,0 +1,21 @@
+"use strict";
+
+export default function rounded(x, y, w, h, cornerRadius) {
+  console.log(cornerRadius)
+  var retval;
+  retval  = "M" + (x + cornerRadius.topLeft) + "," + y;
+  retval += "h" + (w - cornerRadius.topLeft - cornerRadius.topRight);
+  if (cornerRadius.topRight != 0) { retval += "a" + cornerRadius.topRight + "," + cornerRadius.topRight + " 0 0 1 " + cornerRadius.topRight + "," + cornerRadius.topRight; }
```

Comment:
These lines are ridiculously long.

At the very least, add a line after line 5 like the following:
`const { topLeft, topRight, bottomLeft, bottomRight } = cornerRadius`
then you can just use `topLeft`, etc, instead of `cornerRadius.topLeft`